packages/app-extensions/src/actions/modules/actionHandlers/legacyAction.js

```diff
@@ -160,6 +163,31 @@ export function* getSelection(selection) {
   return legacySelection
 }
 
+export function* getScope() {
+  const list = yield select(entityListSelector)
+  const detail = yield select(entityDetailSelector)
+
+  if (list && detail) {
```

Comment:
could this ever be the case? :)

Reply:
I don't hope so :) but it's not impossible and I don't like debugging bugs that occur because of problems like that :)